tests/test_cognitoidp/test_cognitoidp.py

```diff
@@ -325,6 +326,7 @@ def test_delete_identity_providers():
 def test_admin_create_user():
     conn = boto3.client("cognito-idp", "us-west-2")
 
+    username = str(uuid.uuid4())
```

Comment:
Any need for the redundant line?

Reply:
Sorry, this is an oversight. I've already deleted it.